### ISSUE TITLE
New incremental snapshot downloads

### ISSUE THREAD

**Issue by kaber2** [maticnetwork/bor#866]
I'm not sure if this is the proper venue to report this, but it doesn't seem completely inappropriate.

So, I was wondering what the idea behind the "incremental snapshot" download (https://snapshots.polygon.technology/) is. A few numbers:

- the base snapshot is 1.4TB
- the daily incremental archives are roughly 100GB each
- the old snapshot was around 1.8TB

So after just four days, the download size already exceeds the size of a normal snapshot. The current total size is 2.9TB.

I'm assuming the idea is not to add new daily incremental archives endlessly, but to replace the base snapshot occasionally. Given the current sizes stated above, after just four days, it is more beneficial to just download a full snapshot.

On top of that, the storage requirements to download a snapshot, which were 1.8TB for the snapshot + 2.1TB for unpacking, so roughly 4TB before, are now 2.9TB + 2.1TB, so 5TB total, and growing daily. 

Last but not least, unpacking almost twice the data obviously also takes around twice the time.

So in conclusion, we have bigger downloads, higher storage requirements and slower unpacking, without any benefit at all, unless you are downloading snapshots more often than every 4 days. So what's the idea behind this? Can we please get normal snapshots back?

**Comment by kaber2** [comment]
Adding one more datapoint:

- the heimdall "incremental snapshot" unpacked takes 346G
- an old heimdall instance running for almost 1.5 years only takes 304G

While I'm not familiar with the haimdall storage layer, I presume the difference comes from it actually deleting files while running, while the incremental snapshots of course can only add files on top.

So I'll add "leaves useless junk on the disk" to my conclusion above.

**Comment by kaber2** [comment]
So final datapoint, same problem for the bor snapshot, currently sitting at 3.1TB and still unpacking.

So this fails on every level and at every possible metric. It baffles me how someone could actually put this junk live. As if Bor doesn't use enough space already.

**Comment by rebelArtists** [comment]
@kaber2 changes now live here that minimize disk requirements: https://wiki.polygon.technology/docs/operate/snapshot-instructions-heimdall-bor/
